account_invoice_supplier_number_info/README.rst

```diff
@@ -0,0 +1,46 @@
+ .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
```

Comment:
Please modify the module title syntax as shown in https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst